tools/sdmpredictors/sdmpredictors_list_layers.xml

```diff
@@ -0,0 +1,200 @@
+<tool id="sdmpredictors_list_layers" name="SdmPredictors List Layers" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="21.05">
+    <description>Terrestrial and marine predictors for species distribution modelling, list climate layers from multiple datasets</description>
```

Comment:
I recommend to keep the description short and crisp. It's displayed in the tool bar and has not so much space

Reply:
Done